modules/cli/src/integrationTest/java/org/apache/ignite/internal/cli/commands/configuration/ItConfigCommandTest.java

```diff
@@ -114,4 +114,63 @@ void updateConfigWithSpecifiedPath() {
                 () -> assertOutputContains("\"idleSyncTimeInterval\" : 2000")
         );
     }
+
+    @Test
+    @DisplayName("Should update config with key-value format when valid cluster-endpoint-url is given")
+    void updateClusterConfigWithoutQuoting() {
+        execute("cluster", "config", "update", "--cluster-endpoint-url", NODE_URL,
+                "security.authentication.providers.basic1={type=basic,username=asd,password=asadf}");
+
+        assertAll(
+                this::assertExitCodeIsZero,
+                this::assertErrOutputIsEmpty,
+                this::assertOutputIsNotEmpty
+        );
+
+
+        //Emulate config with spaces
+        execute("cluster", "config", "update", "--cluster-endpoint-url", NODE_URL,
+                "security.authentication.providers.basic1", "=", "{", "type=basic,", "username=asd,", "password=asadf}");
+
+        assertAll(
+                this::assertExitCodeIsZero,
+                this::assertErrOutputIsEmpty,
+                this::assertOutputIsNotEmpty
+        );
+
+
+    }
+
+    @Test
+    void updateClusterWithQuotedArgs() {
```

Comment:
Can you add a case when one string is not working and the quoted one is working? For example, some value is `--verbose`.